docs/contributing.md

```diff
@@ -35,12 +35,12 @@ To merge a pull request, the following is necessary:
 Pull requests with changes that are visible to our end users should always include one (or more) release note describing the changes.
 For other changes, these release notes are optional, but make sure to set the flag `show_in_stores` to false.
 
-To add a release note:
-* Have a look at the [template](../release-notes/ReleaseNoteTemplate.yml).
-* Add a new release note to the [unreleased directory](../release-notes/unreleased).
+To add a release note (exemplary on native):
+* Have a look at the [template](native/release-notes/ReleaseNoteTemplate.yml).
+* Add a new release note to the [unreleased directory](native/release-notes/unreleased).
```

Comment:
I think we only have one release notes dir in the root of the monorepo, don't we?